core/Enclave/enclave_hsm.cpp

```diff
@@ -820,70 +718,79 @@ sgx_status_t enclave_get_rand(uint8_t *data, uint32_t datalen)
 }
 
 sgx_status_t enclave_generate_apikey(sgx_ra_context_t context,
-                                     uint8_t *p_apikey, uint32_t apikey_len,
-                                     uint8_t *cipherapikey, uint32_t cipherapikey_len)
+                                     uint8_t *p_apikey, uint32_t apikey_size,
```

Comment:
actually i mean to align the uint32_t to size_t.......not for the parameter name